exercises/pythagorean-triplet/src/example.c

```diff
@@ -0,0 +1,56 @@
+#include "pythagorean_triplet.h"
+
+static triplets_t *no_triplets(void)
+{
+   triplets_t *triplets = malloc(sizeof(triplets_t));
+
+   if (triplets)
+      triplets->count = 0;
+
+   return triplets;
+}
+
+static triplets_t *add_triplet(triplets_t * triplets, uint16_t a, uint16_t b,
+                               uint16_t c)
+{
+   size_t count = 0;
+   if (triplets)
+      count = triplets->count;
+
+   triplets_t *new_triplets = realloc(triplets, sizeof(triplets_t) +
+                                      (sizeof(uint16_t) * LENGTH_OF_TRIPLET) *
+                                      ++count);
```

Comment:
size calculation looks icky, any suggestions?

Reply:
Use a struct for the triplet :)